recipes/raxml-ng/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# pthreads
+mkdir build_pthreads
+pushd build_pthreads
+   cmake ..
+   make
+   install -d ${PREFIX}/bin
+   install ../bin/raxml-ng ${PREFIX}/bin
+popd
+
+# mpi
+mkdir build_mpi
+pushd build_mpi
+   cmake -DUSE_MPI=ON ..
```

Comment:
Is there an openmpi that's getting pulled in?

Reply:
Have added openmpi to host section now, but maybe I should be putting this in run (will check tomorrow)

Reply:
Yeah, it should be in `run:` then too.

Reply:
I expect you'll need `gmp` in the run section as well.

Reply:
I checked the executables with `ldd`, looks like `openmpi` is needed at run but not `gmp`.